src/libs/utils.js

```diff
@@ -33,6 +33,26 @@ export const UserProperties = {
   ZIPCODE: "zipcode"
 };
 
+export const nonCancellableCollectionStatuses = ['Canceled', 'Under Election'];
+
+export const determineCollectionStatus = (collection) => {
+  const { electionMap } = collection;
+  return !isEmpty(electionMap) ?
+    'Under Election' :
+    isCollectionCanceled(collection) ? 'Canceled' : 'Submitted';
+};
+
+export const isCollectionCanceled = (collection) => {
+  const { dars } = collection;
+  return every((dar) => toLower(dar.data.status) === 'canceled')(dars);
+};
```

Comment:
These, too, are good candidates for pulling into a utility method in `DarCollection` in consent, but this works here just fine. It might be nice to group them under an umbrella constant, i.e.
```
export const DarCollection = {
    determineCollectionStatus: (collection) => { blah },
    isCollectionCanceled: (collection) => { blah }
}
```

Reply:
Yeah, `utils,js` is getting a little to freeform with the functions that I keep adding. Grouping them can make the file easier to read through.